code/Attack.ipynb

```diff
@@ -0,0 +1,1620 @@
+{
```

Comment:
<p>Before starting the cell, provide a brief description of what the readers will be doing in this notebook. Considering including any important details and references. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>